lib/elixir/lib/gen_server.ex

```diff
@@ -159,6 +159,36 @@ defmodule GenServer do
   the GenServer callbacks as doing so will cause the GenServer to misbehave.
   If you want to receive custom messages, always receive them in `handle_info/2`.
 
+  Custom messages are all other messages a server may receive that are not sent via
+  `call/3` or `cast/2`.
+
+  To send custom message use: `Kernel.send/2`,`Process.send_after/3` functions.
```

Comment:
From my point of view such short statements are not expressive enough. Someone can easily skip part in brackets when reading.
So I'd prefer to keep two these sentences separately.